frontend/src/modules/Dashboard/Styles/GroupCard.style.tsx

```diff
@@ -1,11 +1,24 @@
 import styled from 'styled-components'
 import { FiClock } from 'react-icons/fi'
+import { ReactComponent as GroupsIcon } from '@assets/img/groupsicon.svg'
+import { ReactComponent as PlusIcon } from '@assets/img/plusicon.svg'
+import { ReactComponent as NewlyMatchable } from '@assets/img/newlymatchable.svg'
 import { colors, h3, h4, StyledComponent } from '@core'
+import { Props } from 'react'
 
-export const StyledContainer = styled.div`
+export const StyledGroupsIcon = styled(GroupsIcon)`
+  padding-right: 1rem;
+`
+
+export const StyledPlusIcon = styled(PlusIcon)`
+  padding-right: 1rem;
+`
+export const StyledNewlyMatchable = styled(NewlyMatchable)``
+
```

Comment:
One thing that I have issue with here is that you're making entirely new components for what seem to be trivial style changes (either one change or none at all), and with MUI v5, I believe the customization would be a lot easier with simple, one-off changes. You don't have MUI v5 in this branch, which I will leave below for more details, but I will leave this here:
https://mui.com/system/the-sx-prop/
https://mui.com/customization/how-to-customize/

Reply:
Yes once you merge in the changes from the MUI v5 migration, small style changes like this should be a lot simpler!

Reply:
Remember to fix this part! especially the empty one